### PR TITLE
Save map zoom and center in map view options

### PR DESCRIPTION
**Pull request by thequeenofspades**
Adds map zoom and center to map view options, so it will be saved to local storage and restored along with other settings.

_*No UI change*_

### CODE REVIEW

File: src/interface/src/app/map/map-manager.ts [hunk 1]
```diff
@@ -39,11 +40,13 @@ export class MapManager {
   constructor(
     private matSnackBar: MatSnackBar,
     maps: Map[],
+    mapViewOptions$: BehaviorSubject<MapViewOptions>,
     popupService: PopupService,
     startLoadingLayerCallback: (layerName: string) => void,
     doneLoadingLayerCallback: (layerName: string) => void
   ) {
     this.maps = maps;
```
Comment (riecke) [inline src/interface/src/app/map/map-manager.ts:48]: Nit: do you need to make these assignments in the constructor?  https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd

File: src/interface/src/app/map/map-manager.ts [hunk 2]
```diff
@@ -341,6 +311,48 @@ export class MapManager {
     });
   }
 
+  private setUpPanHandler(map: L.Map) {
```
Comment (riecke) [inline src/interface/src/app/map/map-manager.ts:314]: I like the name of the function!

File: src/interface/src/app/map/map-manager.ts [hunk 3]
```diff
@@ -39,11 +40,13 @@ export class MapManager {
   constructor(
     private matSnackBar: MatSnackBar,
     maps: Map[],
+    mapViewOptions$: BehaviorSubject<MapViewOptions>,
     popupService: PopupService,
     startLoadingLayerCallback: (layerName: string) => void,
     doneLoadingLayerCallback: (layerName: string) => void
   ) {
```
Comment (riecke) [inline src/interface/src/app/map/map-manager.ts:47]: Nit: do you need to make these assignments in the constructor? https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd

**Review by riecke — APPROVED** [review]

File: src/interface/src/app/map/map-manager.ts (hunk 3 above)
Comment (thequeenofspades) [reply src/interface/src/app/map/map-manager.ts]: Good point!

**Pull request merged**